joomla/joomla-cms
# [5.1] Use Dialog for Plugin editing

### Summary of Changes

Use Dialog for Plugin editing modals.
Similar to https://github.com/joomla/joomla-cms/pull/42423


### Testing Instructions
Run `npm install`.
Disable plugins:
Content - Smart Search
System - Redirect

Visit Finder Index overview, you should see a message about disable pplugin, with link to edit it. Try to use it for change plugin state.

Visit Redirects overview, you should see a message about disable pplugin, with link to edit it. Try to use it for change plugin state.



### Actual result BEFORE applying this Pull Request
Works with BS modal


### Expected result AFTER applying this Pull Request
Works with Joomla Dialog


### Link to documentations
Please select:
- [x] Pull Request link for manual.joomla.org: https://github.com/joomla/Manual/pull/208

References:
- https://github.com/joomla/joomla-cms/pull/40150
